### PR TITLE
feat(test-app): update test-app to hybrid editor and add ability to configure ctfl env vars via localStorage []

### PR DESCRIPTION
**Pull request by elylucas**
Note:

The env vars were renamed. Setup your local .env.local file with the following vars:

VITE_CTFL_ENVIRONMENT=master
VITE_CTFL_SPACE=....
VITE_CTFL_ACCESS_TOKEN=....
VITE_CTFL_PREVIEW_ACCESS_TOKEN=....
VITE_CTFL_EXPERIENCE_TYPE=devLayout
VITE_CTFL_DOMAIN=flinkly.com

### CODE REVIEW

File: packages/test-app/src/App.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import { createBrowserRouter, RouterProvider } from 'react-router-dom';
 import Page from './Page';
+// import SpaceSelector from './components/SpaceSelector';
```
Comment (elylucas) [inline packages/test-app/src/App.tsx:3]: left commented out on purpose for future work

File: packages/test-app/src/components/SpaceForm.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
```
Comment (elylucas) [inline packages/test-app/src/components/SpaceForm.tsx:1]: These Space components aren't currently used, but will be the UI for configuring spaces in the future

File: packages/test-app/src/hooks/useFetchBySlug.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { ExternalSDKMode, useFetchExperience } from '@contentful/experience-builder';
```
Comment (elylucas) [inline packages/test-app/src/hooks/useFetchBySlug.ts:1]: This hook is temporary and will be taken out once the PR to merge in the `fetchBySlug` hook is in for the sdk

File: packages/test-app/src/components/SpaceForm.tsx (hunk 2 above)
Comment (chasepoirier) [reply packages/test-app/src/components/SpaceForm.tsx:1]: Nice!

**Review by chasepoirier — APPROVED** [review]

**Comment by chasepoirier** [comment]
@elylucas Heads up, just modified the base branch to `development` for this PR

**Pull request merged**